tests/test_computeExposureSummaryStats.py

```diff
@@ -43,11 +45,22 @@ def testComputeExposureSummary(self):
         np.random.seed(12345)
 
         # Make an exposure with a noise image
+        exposure = afwImage.ExposureF(100, 100)
+
         band = "i"
         physical_filter = "test-i"
-        exposure = afwImage.ExposureF(100, 100)
```

Comment:
Why relocate this line?

Reply:
It makes more sense to me to define the exposure metadata prior to creating the exposure, but really this is a relic of a lot of testing.

Reply:
Yeah, it obviously doesn't really matter, just making a more general point about trying to have clean diffs really.

Reply:
Point taken (and this change is really a toss up; though I do prefer it this way).